server/src/main/scala/actors/FixedPointsActorBase.scala

```diff
@@ -101,8 +102,9 @@ class FixedPointsActorBase(now: () => SDateLike) extends RecoveryActorLike with
       log.info(s"Received request to snapshot")
       takeSnapshot(stateToMessage)
 
-    case u =>
-      log.info(s"unhandled message: $u")
+    case StreamCompleted => log.warn("Received shutdown")
```

Comment:
StreamFailure here too maybe? Not sure if it would be relevant

Reply:
Same - this actor won't get any failure messages